charts/k8s-monitoring/values.yaml

```diff
@@ -1508,27 +1508,45 @@ profiles:
     # @section -- Profiles (eBPF)
     demangle: none
 
-  # TBD
-  # java:
-  #  enabled: true
-  #  namespaces: []
-  #  # -- Rule blocks to be added to the discovery.relabel component for Java profile sources.
-  # ([docs](https://grafana.com/docs/alloy/latest/reference/components/discovery.relabel/#rule-block))
-  #  extraRelabelingRules: ""
+  java:
+    # -- Gather profiles by scraping java HTTP endpoints
+    # @section -- Profiles (java)
+    enabled: true
```

Comment:
Just to make sure I am getting this right: Profile collecting is by default off, but once `profiles.enabled: true` the java scraping will be enabled.

Reply:
Yep! It all gets enabled when the user sets:
```
profiles:
  enabled: true
```